treebeard.yaml

```diff
@@ -0,0 +1,20 @@
+cell_execution_timeout_seconds: 600
+notebooks:
+  - teachopencadd/000_*/talktorial.ipynb
+  - teachopencadd/001_*/talktorial.ipynb
+  - teachopencadd/002_*/talktorial.ipynb
+  - teachopencadd/003_*/talktorial.ipynb
+  - teachopencadd/004_*/talktorial.ipynb
+  - teachopencadd/005_*/talktorial.ipynb
+  - teachopencadd/006_*/talktorial.ipynb
+  - teachopencadd/007_*/talktorial.ipynb
+  - teachopencadd/008_*/talktorial.ipynb
+  - teachopencadd/009_*/talktorial.ipynb
+  - teachopencadd/010_*/talktorial.ipynb
+  - teachopencadd/011_*/talktorial.ipynb
+  - teachopencadd/012_*/talktorial.ipynb
+  - teachopencadd/013_*/talktorial.ipynb
+  - teachopencadd/014_*/talktorial.ipynb
+  - teachopencadd/015_*/talktorial.ipynb
+  - teachopencadd/016_*/talktorial.ipynb
+  - teachopencadd/017_*/talktorial.ipynb
```

Comment:
This is just beautiful - short and clean :)

Reply:
There's an autodiscovery option we will enable later, but for now we are specifying each one individually. The plan is to obtain a broad view of the state of each notebook's "testability", and then go one by one fixing as needed.

Reply:
Even better!